### ISSUE TITLE
Question: Why are they hardcoded string in the configs?

### ISSUE THREAD

**Issue by CjHare** [windranger-io/windranger-treasury-subgraph#1]
Do these addresses need manual updating when the contracts are deployed/redeployed?

e.g. https://github.com/windranger-io/windranger-treasury-subgraph/blob/main/config/mainnet.json

**Comment by grezle** [comment]
Yes, these feed the `yarn prepare:*` script and are used to generate chain specific versions of `subgraph.yaml`

**Comment by CjHare** [comment]
Alright, now I know that Matchstick is mocking the contract, these hardcoded value simply need to match those from the Matchstick contract?

Hardcoding is likely fine, perhaps putting the TypeScript references into a single constant then a comment above along the lines:
```
// Must match entry in JSON config file
```

wdyt?
